src/components/AppSidebar/AppSidebar.vue

```diff
@@ -31,34 +31,22 @@
 	### Standard usage
 
 	```vue
-	<template>
-		<AppSidebar
-			:title="title"
-			:subtitle="subtitle">
-			<!-- Insert your slots and tabs here -->
-		</AppSidebar>
-	</template>
-	<script>
-		export default {
-			data() {
-				return {
-					title: 'cat-picture.jpg',
-					subtitle: 'last edited 3 weeks ago'
-				}
-			}
-		}
-	</script>
+	<AppSidebar
+		title="cat-picture.jpg"
+		subtitle="last edited 3 weeks ago">
+		<!-- Insert your slots and tabs here -->
+	</AppSidebar>
 	```
 
 	### Editable title
+
 	```vue
 	<template>
 		<AppSidebar
-			:title="title"
+			:title.sync="title"
```

Comment:
This change is also really unrelated to the actual aim of this PR, no?

Reply:
Nevermind, it's only in the docs, so who cares 😄🤷

Reply:
It is fixing the actual docs yep.